### DISCUSSION TITLE
请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！

### DISCUSSION THREAD

**Discussion by BRUCELVCHINA** [getlantern/lantern-forum-cn#180] [Q&A]
请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！

**Comment by MaxMKMax** [comment]
@BRUCELVCHINA 您好, 请问您可以向我们的支付团队发送电子邮件吗？因为我们想保护您的个人信息和隐私。因此，最好直接向我们的支付团队发送电子邮件申请退款。

电子邮件：support@getlantern.org

**Comment by BRUCELVCHINA** [comment]
为什么还没退款？发送给支付团队的邮件被屏蔽了



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mike K ***@***.***&gt;
发送时间: 2024年2月27日 00:45
收件人: getlantern/lantern-forum-cn ***@***.***&gt;
抄送: BRUCELVCHINA ***@***.***&gt;, Mention ***@***.***&gt;
主题: Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@BRUCELVCHINA 您好, 请问您可以向我们的支付团队发送电子邮件吗？因为我们想保护您的个人信息和隐私。因此，最好直接向我们的支付团队发送电子邮件申请退款。
 
***@***.***

**Reply by MaxMKMax** [reply]
@BRUCELVCHINA 
support@getlantern.org
您能看到这个邮箱吗?

**Reply by MaxMKMax** [reply]
@BRUCELVCHINA 需要您直接发给我们的支付团队来进行退款.

**Comment by BRUCELVCHINA** [comment]
发了邮件，收到的是这些。为什么？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月28日(星期三) 晚上11:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@BRUCELVCHINA
 ***@***.***
 您能看到这个邮箱吗?

**Reply by MaxMKMax** [reply]
@BRUCELVCHINA 只要您发送电子邮件至support @ getlantern.org，我们的付款团队将为您办理退款。但这需要几个工作日。感谢您的等待，并希望将来再次见到您! 

*电子邮件之间不包含空格。这样写可以避免github隐藏完整的邮件并且你可以清楚地看到它。

**Comment by BRUCELVCHINA** [comment]
回复后，显示如下信息



Message blocked
Your message ***@***.***&nbsp;has been blocked. See technical details below for more information.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月1日(星期五) 凌晨0:46
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@BRUCELVCHINA 只要您发送电子邮件至support @ getlantern.org，我们的付款团队将为您办理退款。但这需要几个工作日。感谢您的等待，并希望将来再次见到您!
 
*电子邮件之间不包含空格。这样写可以避免github隐藏完整的邮件并且你可以清楚地看到它。

**Reply by MaxMKMax** [reply]
@BRUCELVCHINA 好的. 明白问题出现在哪里了. 给我们一点时间弄一下. 也会帮您处理您的事情. 弄好了再通知您.

**Comment by BRUCELVCHINA** [comment]
请问什么时候处理好？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月1日(星期五) 凌晨2:41
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@BRUCELVCHINA 好的. 明白问题出现在哪里了. 给我们一点时间弄一下. 也会帮您处理您的事情. 弄好了再通知您.

**Reply by MaxMKMax** [reply]
@BRUCELVCHINA 你好。您可以直接给我您购买时的电子邮件吗？我直接要求付款团队手动为您办理退款。

**Comment by BRUCELVCHINA** [comment]
怎么样了？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月5日(星期二) 晚上10:25
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@BRUCELVCHINA 你好。您可以直接给我您购买时的电子邮件吗？我直接要求付款团队手动为您办理退款。

**Reply by MaxMKMax** [reply]
@BRUCELVCHINA 请向我提供您注册专业版的电子邮件。否则我无法帮助您处理退款请求。谢谢。

**Reply by MaxMKMax** [reply]
@BRUCELVCHINA 这里的客服和支付团队的客服是不一样的. 所以我无法知道您是哪个用户.

**Comment by BRUCELVCHINA** [comment]
***@***.***




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月14日(星期四) 凌晨0:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@BRUCELVCHINA 请向我提供您注册专业版的电子邮件。否则我无法帮助您处理退款请求。谢谢。

**Reply by MaxMKMax** [reply]
@BRUCELVCHINA 您好. 您方便截个图您的邮件发到这里吗? GitHub屏蔽了您写的邮箱. 我看得的是这样 xxxx@xxxxx. 谢谢!

**Comment by MaxMKMax** [comment]
<img width="129" alt="image" src="https://github.com/getlantern/lantern-forum-cn/assets/155475142/5eb30f04-cd2e-4f0f-bb6b-a4778dd9298f">

可以这么截个图.

**Comment by BRUCELVCHINA** [comment]
请问还没有处理么？


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月14日(星期四) 晚上9:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 image.png (view on web) 
可以这么截个图.

**Reply by MaxMKMax** [reply]
@BRUCELVCHINA 您好. 您还没有提供购买单据和邮箱给我. 无法帮您申请呢.
我不是客服部门或者支付部门的. 无法直接找您的信息.

**Comment by BRUCELVCHINA** [comment]
不是给你截图了么




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月25日(星期一) 凌晨0:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@BRUCELVCHINA 您好. 您还没有提供购买单据和邮箱给我. 无法帮您申请呢.
 我不是客服部门或者支付部门的. 无法直接找您的信息.

**Reply by MaxMKMax** [reply]
@BRUCELVCHINA 您好. 我这看不到. 完全看不到呢. 还是您直接发截图到我的邮件给我吧! 
mike@getlantern.org

**Comment by BRUCELVCHINA** [comment]
什么时候退费，你们的VPN很不好用！！！




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月25日(星期一) 凌晨0:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@BRUCELVCHINA 您好. 您还没有提供购买单据和邮箱给我. 无法帮您申请呢.
 我不是客服部门或者支付部门的. 无法直接找您的信息.

**Comment by BRUCELVCHINA** [comment]
已经发你们截图了，为什么还不处理？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月25日(星期一) 晚上9:47
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@BRUCELVCHINA 您好. 我这看不到. 完全看不到呢. 还是您直接发截图到我的邮件给我吧!
 ***@***.***

**Comment by BRUCELVCHINA** [comment]
已经发你们截图了，为什么还不处理？




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月25日(星期一) 晚上9:47
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@BRUCELVCHINA 您好. 我这看不到. 完全看不到呢. 还是您直接发截图到我的邮件给我吧!
 ***@***.***

**Reply by MaxMKMax** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/afbe36f2-8245-4012-9641-a21f513999b9)


@BRUCELVCHINA 您好. 我们这里完全看不到您的截图和信息. 之前让您重发了但您没给我们重发. 您的信息都被GitHub屏蔽了. 您要不直接发邮件给我吧, 我另外帮您提交上去申请退款. 需要您的账号邮箱和付款单据. 谢谢!
mike@getlantern.org

**Comment by MaxMKMax** [comment]
@BRUCELVCHINA 
您给的截图都是没有任何信息的. 连邮箱也看不到啊. 您发邮件给我了再过来通知我.
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/0e385b03-17ff-4717-afe1-a66036993645)

**Comment by BRUCELVCHINA** [comment]
邮箱账号及付款单据都在附件中，请查收！




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月24日(星期三) 下午5:16
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
image.png (view on web)
 
@BRUCELVCHINA 您好. 我们这里完全看不到您的截图和信息. 之前让您重发了但您没给我们重发. 您的信息都被GitHub屏蔽了. 您要不直接发邮件给我吧, 我另外帮您提交上去申请退款. 需要您的账号邮箱和付款单据. 谢谢!
 ***@***.***

**Comment by MaxMKMax** [comment]
@BRUCELVCHINA 您好. 您使用哪一个邮件发的? 是发给support@getlantern.org? 还是mike@getlantern.org? 谢谢.

**Comment by BRUCELVCHINA** [comment]
刚刚都转发了



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mike K ***@***.***&gt;
发送时间: 2024年5月7日 00:36
收件人: getlantern/lantern-forum-cn ***@***.***&gt;
抄送: BRUCELVCHINA ***@***.***&gt;, Mention ***@***.***&gt;
主题: Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@BRUCELVCHINA 您好. 您使用哪一个邮件发的? ***@***.***? ***@***.***? 谢谢.

**Reply by MaxMKMax** [reply]
@BRUCELVCHINA 好的. 谢谢!

**Comment by Join-7** [comment]
申请退款马上两个月了，还没有退款处理。

**Reply by MaxMKMax** [reply]
@Join-7 您好. 照道理应该已经完成退款了. 您是哪个邮箱? 请您重新给我您的账号邮箱和支付单据. 我帮您问一下.

**Comment by BRUCELVCHINA** [comment]
已经发过很多遍了。



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mike K ***@***.***&gt;
发送时间: 2024年6月12日 23:45
收件人: getlantern/lantern-forum-cn ***@***.***&gt;
抄送: BRUCELVCHINA ***@***.***&gt;, Mention ***@***.***&gt;
主题: Re: [getlantern/lantern-forum-cn] 请退款，只能在电脑上用，不知道在手机ipad如何应用专业版，学习成本太高！ (Discussion #180)





 
@Join-7 您好. 照道理应该已经完成退款了. 您是哪个邮箱? 请您重新给我您的账号邮箱和支付单据. 我帮您问一下.
